website/contributing/contributing-intro.md

```diff
@@ -0,0 +1,113 @@
+2. **Reviewing pull requests for the docs.** Reviewing [documentation updates](https://github.com/facebook/react-native-website/pulls) can be as simple as checking for spelling and grammar. If you encounter situations that can be explained better in the docs, click **Edit** at the top of most docs pages to get started with your own contribution.
+3. **Help people write test plans.** Some pull requests sent to the main repository may lack a proper test plan. These help reviewers understand how the change was tested, and can speed up the time it takes for a contribution to be accepted.
+
+Each of these tasks is highly impactful, and maintainers will greatly appreciate your help.
+
+### Our Development Process
+
+We use GitHub issues and pull requests to keep track of bug reports and contributions from the community. All changes from engineers at Meta will sync to [GitHub](https://github.com/facebook/react-native) through a bridge with Meta's internal source control. Changes from the community are handled through GitHub pull requests. 
+
+Once a change made on GitHub is approved, it will first be imported into Facebook's internal source control and tested against Facebook's codebase. Once merged at Facebook, the change will eventually sync back to GitHub as a single commit once it has passed Facebook's internal tests.
+
+You can learn more about the contribution process in the following documents:
+
+- [Triaging GitHub Issues](/contributing/triaging-github-issues)
+- [Managing Pull Requests](/contributing/managing-pull-requests)
+
+We also have a thriving community of contributors who would be happy to help you get set up. You can reach out to us through [@ReactNative](http://twitter.com/reactnative) (the React Native team) and [@ReactNativeComm](http://twitter.com/reactnativecomm) (the React Native Community organization).
+
+### Repositories
+
+The main repository contains the React Native framework itself, and it is here where we keep track of bug reports and manage pull requests.
+
+There are a few other repositories you might want to familiarize yourself with:
+
+- **React Native website** which contains the source code for the website, including the documentation, located [in this repository](https://github.com/facebook/react-native-website).
+- **Releases** Conversations for new releases are happening [in this discussion repo](https://github.com/reactwg/react-native-releases/discussions).
+- **Changelog** The changelog can be found [here](https://github.com/facebook/react-native/blob/main/CHANGELOG.md).
+- **Discussions** about the future of React Native take place in the <https://github.com/react-native-community/discussions-and-proposals> repository.
+- **High-quality plugins** for React Native can be found throughout the [React Native Community GitHub Organization](http://github.com/react-native-community/).
```

Comment:
@Simek the community plugins have been moved out into separate organizations and projects, so this line is no longer relevant

Reply:
```suggestion
- **High-quality plugins** for React Native can be found throughout the [React Native Directory](https://reactnative.directory) website.
```

@cortinico What do you think about adding mention for the RN Directory here instead?